src/server/middleware/authentication.ts

```diff
@@ -0,0 +1,108 @@
+import passport from 'passport'
+import passportLocal, { VerifyFunctionWithRequest } from 'passport-local'
+import bcrypt from 'bcryptjs'
+import { Express } from 'express'
+import { FieldValidators, User, UserStatus, Validator } from '@openforis/arena-core'
+
+import { ExpressInitializer } from '../expressInitializer'
+import { UserServiceServer } from '../../service'
+import { UserResetPasswordRepository, AuthGroupRepository } from '../../repository'
+
+const comparePassword = bcrypt.compareSync
+
+// TODO: Handle these separately
```

Comment:
what is this TODO?

Reply:
Should we keep those messages here or move them to core?

Reply:
I didn't understand what is the TODO

Reply:
```
// TODO: Handle these separately
const ValidationMessages = {
  emailInvalid: 'validationErrors.user.emailInvalid',
  userNotFound: 'validationErrors.user.userNotFound',
  passwordChangeRequired: 'validationErrors.user.passwordChangeRequired',
}
```

Where should these be?